simplq/src/styles/homePage.module.scss

```diff
@@ -69,7 +69,131 @@ div {
     display: flex;
     align-items: center;
     p {
-      min-width: 10rem; // So that the text alings along a vertical line, as the entre component in vertically centered
+      min-width: 10rem; // So that the text aligns along a vertical line, as the entre component in vertically centered
     }
   }
 }
+
+.footer {
+  @include background-color();
+  @include text-color();
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+}
+
+.main-card-container {
+  padding: 0;
+  margin: 0;
+  width: 60%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 5%;
+}
+
+.card {
+  flex: 1;
+  margin: 10px;
+  padding: 5px;
+  min-width: 200px;
+  max-width: 300px;
+  display: flex;
```

Comment:
Can you stick to rem instead of px everywhere except viewport relative measurements?

Reply:
I changed margin px's and padding px's to rem's